cernopendata/modules/fixtures/data/records/cms-derived-Run2-datascience.json

```diff
@@ -0,0 +1,386 @@
+      "type": "UInt_t",
+      "variable": "eventJetMult"
+    },
+    {
+      "description": "Indicates the ranking number of the jet, as the jets are ordered by their transverse momenta within a single event.",
+      "type": "UInt_t",
+      "variable": "jetPtOrder (UInt_t)"
+    },
+    {
+      "description": "The phi difference of the two leading jets.",
+      "type": "Float_t",
+      "variable": "dPhiJetsLO"
+    },
+    {
+      "description": "The eta difference of the two leading jets.",
+      "type": "Float_t",
+      "variable": "dEtaJetsLO"
+    },
+    {
+      "description": "If there are at least 3 jets in the event, alpha is the third jet's transverse momentum divided by the average transverse momentum of the two leading jets.",
+      "type": "Float_t",
+      "variable": "alpha"
+    },
+    {
+      "description": "Event number.",
+      "type": "ULong64_t",
+      "variable": "event"
+    },
+    {
+      "description": "Run number.",
+      "type": "UInt_t",
+      "variable": "run"
+    },
+    {
+      "description": "Luminosity block.",
+      "type": "UInt_t",
+      "variable": "lumi"
+    },
+    {
+      "description": "Transverse momentum of the generated hard process.",
+      "type": "Float_t",
+      "variable": "pthat"
+    },
+    {
+      "description": "Weight assigned to the generated event.",
+      "type": "Float_t",
+      "variable": "eventWeight"
+    },
+    {
+      "description": "The median density (in GeV/A) of pile-up contamination per event; computed from all PF candidates of the event.",
+      "type": "Float_t",
+      "variable": "rhoAll"
+    },
+    {
+      "description": "Same as above, computed from all PF candidates with |η| < 2.5.",
+      "type": "Float_t",
+      "variable": "rhoCentral"
+    },
+    {
+      "description": "Same as above, computed from all neutral PF candidates with |η| < 2.5.",
+      "type": "Float_t",
+      "variable": "rhoCentralNeutral"
+    },
+    {
+      "description": "Same as above, computed from all PF charged hadrons associated to pileup vertices and with |η| < 2.5.",
+      "type": "Float_t",
+      "variable": "rhoCentralChargedPileUp"
+    },
+    {
+      "description": "The number of good reconstructed primary vertices.",
+      "type": "UInt_t",
+      "variable": "PV_npvsGood"
+    },
+    {
+      "description": "The number of pileup interactions that have been added to the event in the current bunch crossing.",
+      "type": "UInt_t",
+      "variable": "Pileup_nPU"
+    },
+    {
+      "description": "The true mean number of the poisson distribution for this event from which the number of interactions in each bunch crossing has been sampled.",
+      "type": "Float_t",
+      "variable": "Pileup_nTrueInt"
+    }
+  ],
+  "date_published": "2019",
+  "distribution": {
+    "formats": [
+      "FIXME"
+    ],
+    "number_events": 11111,
+    "number_files": 2222,
+    "size": 3333
```

Comment:
number of files: 244

total size: 204611954128

formats: root, h5

Reply:
Four index files should be attached, here are the data:

path=root://eospublic.cern.ch//eos/opendata/cms/datascience/JetNTuple_QCD_RunII_13TeV_MC/file-indexes/CMS_JetNTuple_QCD_RunII_13TeV_MC_h5_file_index.json size=32558 checksum=4d096eae
path=root://eospublic.cern.ch//eos/opendata/cms/datascience/JetNTuple_QCD_RunII_13TeV_MC/file-indexes/CMS_JetNTuple_QCD_RunII_13TeV_MC_h5_file_index.txt size=15020 checksum=7d368ee3
path=root://eospublic.cern.ch//eos/opendata/cms/datascience/JetNTuple_QCD_RunII_13TeV_MC/file-indexes/CMS_JetNTuple_QCD_RunII_13TeV_MC_root_file_index.json size=32964 checksum=b89578d0
path=root://eospublic.cern.ch//eos/opendata/cms/datascience/JetNTuple_QCD_RunII_13TeV_MC/file-indexes/CMS_JetNTuple_QCD_RunII_13TeV_MC_root_file_index.txt size=15264 checksum=c1e21b88